velox/experimental/cudf/exec/Utilities.cpp

```diff
@@ -112,11 +114,24 @@ std::unique_ptr<cudf::table> concatenateTables(
       tableViews, stream, cudf::get_current_device_resource_ref());
 }
 
+std::unique_ptr<cudf::table> makeEmptyTable(TypePtr const& inputType) {
+  std::vector<std::unique_ptr<cudf::column>> emptyColumns;
+  for (size_t i = 0; i < inputType->size(); ++i) {
+    auto emptyColumn = cudf::make_empty_column(
+        cudf_velox::veloxToCudfTypeId(inputType->childAt(i)));
+    emptyColumns.push_back(std::move(emptyColumn));
+  }
+  return std::make_unique<cudf::table>(std::move(emptyColumns));
+}
+
 std::unique_ptr<cudf::table> getConcatenatedTable(
     std::vector<CudfVectorPtr>& tables,
+    TypePtr const& tableType,
```

Comment:
@shrshi Why not "const TypePtr&" ?

Reply:
cudf preferred use of "east const". 
I see that velox use "west const". This will be updated to west const.